### ISSUE TITLE
pause and resume not working

### ISSUE THREAD

**Issue by priyankJksol** [wysaid/android-gpuimage-plus#470]
Pause and resume not working properly in CameraRecordGLSurfaceView

**Comment by wysaid** [comment]
Yes, you can save your video clip when you pause, and start another video clip if resume.
You can combine the all video clips at last.
